src/css/sumo.scss

```diff
@@ -227,6 +227,11 @@ html[data-theme='light'] {
   min-width: 100px;
 }
 
+/* Example: Force global styles back to normal */
+body, .MuiTypography-root, button, input, select, textarea {
+  font-family: 'Lab Grotesque', sans-serif !important;
+}
```

Comment:
**Berry Team**: I added this because the widget code in `src/pages/index.tsx` corrupted some existing CSS, so this override corrects it.